features/evidence_hub_search.feature

```diff
@@ -84,6 +84,18 @@ Feature: Evidence Hub Search
       | year of publication | 2017                                                               |
     And I should see the "first" evidence summary icon linking to "Insight" article
 
+  Scenario: Search by Evidence hub type type
+    When I search the evidence hub for summaries of type evaluation
+    Then I should see "1" evidence summary
+    And I should see the "first" evidence summary as
+      | Field               | Value                       |
+      | document title      | Looking after the pennies   |
+      | evidence type       | Evaluation                  |
+      | topics              | Budgeting and keeping track |
+      | countries           | United Kingdom              |
+      | year of publication | 2017                        |
+    And I should see the "first" evidence summary icon linking to "Evaluation" article
+
```

Comment:
I think is worth to add a search by insight, review or two altogether. Wdyt?

Reply:
Sure! Going to add it.